source/Keystone/ApplicationCredentials.rst

```diff
@@ -0,0 +1,179 @@
+
+To create a new application credential, you can use the command:
+
+.. code-block:: bash
+
+  openstack application credential create [-h]
+                                               [-f {json,shell,table,value,yaml}]
+                                               [-c COLUMN] [--noindent]
+                                               [--prefix PREFIX]
+                                               [--max-width <integer>]
+                                               [--fit-width] [--print-empty]
+                                               [--secret <secret>]
+                                               [--role <role>]
+                                               [--expiration <expiration>]
+                                               [--description <description>]
+                                               [--unrestricted] [-:-restricted]
+                                               <name>
+
+Below is an example of creating an application credential which expires on 06/07/2020 at 00:00:00 and the secret has been generated automatically by OpenStack.
+
+
+.. code-block:: bash
+
+  openstack application credential create --expiration 2020-07-08T00:00:00 --description "Example Application Credential" Example-Credential
+
+  +--------------+----------------------------------------------------------------------------------------+
+  | Field        | Value                                                                                  |
+  +--------------+----------------------------------------------------------------------------------------+
+  | description  | Example Application Credential                                                         |
+  | expires_at   | 2020-07-08T00:00:00.000000                                                             |
+  | id           | APPLICATION_CREDENTIAL_ID                                                              |
+  | name         | Example-Credential                                                                     |
+  | project_id   | PROJECT_ID                                                                             |
+  | roles        | user                                                                                   |
+  | secret       | SECRET                                                                                 |
+  | system       | None                                                                                   |
+  | unrestricted | False                                                                                  |
+  | user_id      | USER_ID                                                                                |
+  +--------------+----------------------------------------------------------------------------------------+
+
+.. note:: 
+
+    The secret is only revealed **once**. If a user has forgotten the secret, a new application credential as to be generated.
+
+g
```

Comment:
Random `g`

Also we should either for a `FAQ` section or drop `Q` / `A` since this doesn't match the tone of the rest of the section